src/attachment/collar/Collar Pose.txt

```diff
@@ -0,0 +1,62 @@
+# OpenCollar Pose Config
+#
+Anim:Bend 1
```

Comment:
Request for changes: Add comment lines indicating what these lines do and what the values mean.